AndelaOSP/wire
# chore: Make URLS environment variables

#### What does this PR do?
Makes the URLS hard-coded in `config/index.js` environment variables. 
#### Where should the reviewer start?
View file changes in PR
#### How should this be manually tested?
Configure your `.env` file `for local development` like this:
```
NODE_ENV=development
API_URL=http://wire.andela.com:8080/api
ANDELA_API_BASE_URL=https://api.andela.com
BASE_URL=http://wire.andela.com:8080
```
The run `yarn start:dev` and everything should still work
